spm12/metadata/get_metadata_val_classic.m

```diff
@@ -123,8 +123,16 @@
         end
         
     case 'EchoTime' % [ms]
-        % Valid for all vendors
+        % Valid for all vendors:
+        % try EchoTime first
         [nFieldFound, fieldList] = find_field_name(mstruc, 'EchoTime', 'caseSens','sensitive','matchType','exact');
+        % TerraX acquisition with new Dicoms chosen:
```

Comment:
This applies to all sequences Kerrin made for the XA software line, I think, not just the TerraX. Perhaps we can generalise the comment to make it more useful?

Reply:
An additional remark: I would guess `EffectiveEchoTime` would also be populated instead of `EchoTime` in segmented sequences.

Reply:
I didn't mean all sequences Kerrin made; TSE is the typical example for the use of this field. I would suggest being more generic, e.g.
```
In some sequences (e.g. the XA implementations of FLASH from Kerrin Pine) the EffectiveEchoTime is populated instead of the EchoTime field in the DICOM.
```